contributing/community/reviews.rst

```diff
@@ -0,0 +1,197 @@
+
+.. topic:: Example
+
+    Here is a sample comment for a bug report that could be reproduced:
+
+    .. code-block:: text
+
+        Thank you @weaverryan for creating this bug report! This indeed looks
+        like a bug. I reproduced the bug in the "kernel-bug" branch of
+        https://github.com/webmozart/symfony-standard.
+
+        Status: Reviewed
+
+The Pull Request Review Process
+-------------------------------
+
+The process for reviewing pull requests (PRs) is similar to doing a review of a
+bug report. Reviews of pull requests usually take a little longer since you need
+to understand the functionality that has been fixed or implemented and then find
+out whether the implementation is complete.
+
+It is okay to do partial reviews. If you do a partial review, comment how far
+you got and leave the PR in "Needs Review" state.
+
+Pick a pull request from the `PRs in need of review`_ and follow these steps:
+
+#. **Is the PR Complete**?
+
+   Every pull request must contain a header that gives some basic information
+   about the PR. You can find the template for that header in the
+   :ref:`Contribution Guidelines <contributing-code-pull-request>`.
+
+#. **Is the Base Branch Correct?**
+
+   GitHub displays the branch that a PR is based on below the title of the
+   pull request. Is that branch correct?
+
+   * Bugs should be fixed in the oldest, maintained version that contains the
+     bug. Check :doc:`Symfony's Release Schedule <releases>` to find the oldest
+     currently supported version.
+
+   * New features should always be added to the current development version.
+     Check the `Symfony Roadmap`_ to find the current development version.
+
+#. **Reproduce the Problem**
+
+   Read the issue that the pull request is supposed to fix. Reproduce the
+   problem on a clean `Symfony Standard Edition`_ project and try to understand
+   why it exists.
+
+#. **Review the Code**
+
+   Read the code of the pull request and check it against some common criteria:
+
+   * Does the code address the issue the PR is intended to fix/implement?
+   * Does the PR stay within scope to address *only* that issue?
+   * Does the PR contain automated tests? Do those tests cover all relevant
+     edge cases?
+   * Does the PR contain sufficient comments to easily understand its code?
+   * Does the code break backwards compatibility? If yes, does the PR header say
+     so?
+   * Does the PR contain deprecations? If yes, does the PR header say so? Does
+     the code contain ``trigger_error()`` statements for all deprecated
+     features?
+   * Are all deprecations and backwards compatibility breaks documented in the
+     latest UPGRADE-X.X.md file? Do those explanations contain "Before"/"After"
+     examples with clear upgrade instructions?
+
+   Eventually, some of these aspects will be checked automatically.
+
+#. **Test the Code**
+
+   Take your project from step 3 and test whether the PR works properly.
+
+   TODO: precise steps
```

Comment:
Probably we don't really need to say much more here anyways. We can probably just remove this TODO line